app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -79,8 +79,7 @@
   <%= validated_form_for :doc_auth,
                          url: url_for,
                          method: 'put',
-                         html: { autocomplete: 'off', class: 'margin-top-2 js-consent-continue-form' } do |f| %>
-    <br/>
```

Comment:
🙏

Reply:
I didn't even see it when I was messing with all of that. I'm not even certain why it was all in there.